art/dashboard/backend.py

```diff
@@ -37,7 +37,7 @@ def prepare_steps_info(logs_path: Path) -> Dict[str, Dict]:
                     "timestamp": run["timestamp"],
                     "hash": run["hash"],
                     "commit_id": run["commit_id"],
-                    "successfull": run["succesfull"],
+                    "successfull": run["successfull"],
```

Comment:
So close, just remove the last "L", there's only one at the and actually

Reply:
xD

Reply:
you are right

Reply:
xD